bindata/assets/alerts/api-usage.yaml

```
@@ -30,7 +30,7 @@ spec:
               a successful upgrade to the next EUS cluster version.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: |
-            group(apiserver_requested_deprecated_apis{removed_release=~"1\\.2[123]"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
+            group(apiserver_requested_deprecated_apis{removed_release=~"1\\.2[234]"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
```

```suggestion
            group(apiserver_requested_deprecated_apis{removed_release=~"1\\.2[5]"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
```
All even-numbered releases are EUS releases now. Going forward this means that for odd-numbered releases `APIRemovedInNextReleaseInUse` and `APIRemovedInNextEUSReleaseInUse` will match.

---

Since the next EUS release is 1.24, shouldn't this still be 1.24 for now? Or if we want to get ahead maybe 1.26 would be better suited, wdyt?

---

Sorry, I was unclear and should of said:

> All even-numbered **OpenShift** releases are EUS releases now.

---

Makes more sense now thanks :)